crates/ruma-events/src/matrix_rtc/focus.rs

```diff
@@ -0,0 +1,85 @@
+//! Types for matrixRTC Focus/SFU configurations.
+
+use serde::{Deserialize, Serialize};
+
+/// Description of the SFU/Focus a membership can be connected to.
+///
+/// A focus can be any server powering the matrixRTC session (SFU,
+/// MCU). It serves as a node to redistribute RTC streams.
+#[derive(Clone, Debug, Serialize, Deserialize)]
+#[cfg_attr(test, derive(PartialEq))]
+#[cfg_attr(not(feature = "unstable-exhaustive-types"), non_exhaustive)]
+#[serde(tag = "type", rename_all = "snake_case")]
+pub enum Focus {
+    /// Livekit is one possible type of SFU/Focus that can be used for a matrixRTC session.
```

Comment:
It seems to me that the spelling is exactly `LiveKit` (with those capital letters). Please make sure that all the docs use that.